### PR TITLE
BIGTOP-3301. Add smoke-tests for hadoop-kms.

### PR DESCRIPTION
**Pull request by iwasakims**

### CODE REVIEW

**Comment by iwasakims** [comment]
minimal config.yaml to test this::
```
docker:
        memory_limit: "12g"
        image: "bigtop/puppet:trunk-centos-7"
distro: centos
components: [hdfs, kms]
enable_local_repo: true
smoke_test_components: [kms]
```
```
$ cd provisioner/docker
$ ./docker-hadoop.sh -c 1 --smoke-tests
```

File: bigtop-tests/test-artifacts/hadoop/src/main/groovy/org/apache/bigtop/itest/hadoop/kms/TestKms.groovy [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.apache.bigtop.itest.shell.Shell;
+
+public class TestKms {
+  private static final String USERNAME = System.getProperty("user.name");
+  private static final String ZONEDIR  = "/user/$USERNAME/zone1";
+  private static final String KEYNAME  = "key1";
+  private static final String LOCALDIR = "temp_kms";
+  private static final String TXTFILE  = "test.txt";
```
Comment (evans-ye) [inline bigtop-tests/test-artifacts/hadoop/src/main/groovy/org/apache/bigtop/itest/hadoop/kms/TestKms.groovy]: Nit-picking: 
```
USER_NAME
ZONE_DIR
KEY_NAME
...
```

Comment (iwasakims) [reply bigtop-tests/test-artifacts/hadoop/src/main/groovy/org/apache/bigtop/itest/hadoop/kms/TestKms.groovy]: fixed.

**Comment by evans-ye** [comment]
+1. Thanks!

**Comment by iwasakims** [comment]
Thanks, @evans-ye.

**Pull request merged**